next-react-query-tailwind/README.md

```diff
@@ -6,20 +6,27 @@ Amplify URL: https://main.d31pj5m17ne70k.amplifyapp.com
 
 ## Table of Contents
 
-- [Overview](#overview)
-  - [Featured Tech Stack](#featured-tech-stack)
-  - [Included Tooling](#included-tooling)
-  - [Demo Application Libraries](#demo-application-libraries)
-- [Getting Started](#getting-started)
-  - [Downloading](#downloading)
-  - [Installation](#installation)
-  - [Generated Files](#generated-files)
-- [Configuration](#configuration)
-  - [Environment Variables](#environment-variables)
-  - [OAuth Setup](#oauth-setup)
-  - [JWT Encryption Keys](#jwt-encryption-keys)
-- [Development](#development)
-- [Deployment](#deployment)
+- [next-react-query-tailwind](#next-react-query-tailwind)
+  - [Table of Contents](#table-of-contents)
```

Comment:
Can you just remove these so everything else doesn't get indented another level. I don't think we need those

Reply:
Updated foor review